developer-hub/src/polkadot_sdk/cumulus.rs

```diff
@@ -0,0 +1,132 @@
+//! # Cumulus
+//!
+//! Substrate provides a framework through which a blockchain node and runtime can easily be
+//! created. Cumulus aims to extend the same approach to creation of Polkadot parachains.
+//!
+//! > Cumulus clouds are shaped sort of like dots; together they form a system that is intricate,
+//! > beautiful and functional.
+//!
+//! ## Example: Runtime
+//!
+//! A cumulus based runtime is fairly similar to a normal [FRAME]-based runtime. Most notably, the
+//! following changes are applied to a normal FRAME-based runtime to make it a cumulus-based
+//! runtime:
+//!
+//! #### Cumulus Pallets
+//!
+//! A parachain runtime should use a number of pallets that are provided by Cumulus. Notably:
+//!
+//! - [`frame-system`], like all FRAME-based runtimes.
+//! - [`cumulus-pallet-parachain-system`]
+//! - [`parachain-info`]
+#![doc = docify::embed!("./src/polkadot_sdk/cumulus.rs", system_pallets)]
+//!
+//! Given that all cumulus-based runtimes use a simple aura-based consensus mechanism, the following
+//! pallets also need to be added:
+//!
+//! - [`pallet-timestamp`]
+//! - [`pallet-aura`]
+//! - [`cumulus-pallet-aura-ext`]
+// #![doc = docify::embed!("./src/lib.rs", consensus_pallets)]
+//!
+//!
+//! Finally, a separate macro, similar to `impl_runtime_api`, which create the default set of
+//! runtime apis, will generate the parachain runtime's main additional runtime api, also known as
+//! PVF.
```

Comment:
Not sure where exactly would be good to link to "official" definition of the PVF (maybe `PvfPrepData `?), but after this is merged we can solve this kind of thing

```suggestion
//! PVF (Parachain Validation Function).
```

Reply:
Please add this to the gloassry as well?

Reply:
The terms backlinking is taking care in the glossary issue https://github.com/paritytech/polkadot-sdk-docs/issues/40